google-cloud-sdk/testSrc/com/google/cloud/tools/intellij/appengine/java/sdk/DefaultCloudSdkVersionNotifierTest.java

```diff
@@ -44,32 +45,41 @@
   @Test
   public void testNotifyIfCloudSdkNotSupported_isSupported() {
     when(cloudSdkValidator.validateCloudSdk()).thenReturn(new HashSet<>());
-    checker.notifyIfUnsupportedVersion();
+    checker.notifyIfVersionOutOfDate();
 
-    verify(checker, times(0)).showNotification();
+    verify(checker, times(0)).showNotification(anyString(), anyString());
```

Comment:
Since `showNotification` might receive different input title and message, could it actually make sense to test against actual strings we expect here?

Reply:
done